### ISSUE TITLE
Too few images: "No good initial image pair found"

### ISSUE THREAD

**Issue by PascalHbr** [NVlabs/instant-ngp#896]
Unfortunately, using the script `python colmap2nerf.py --colmap_matcher exhaustive --run_colmap --aabb_scale 16 --images  data/nerf/custom_images` I receive the following outcome:

`Finding good initial image pair
==============================================================================

 => No good initial image pair found.

Elapsed time: 0.376 [minutes]
ERROR: failed to create sparse model
FATAL: command failed`

I assume, the reason for this is that I only have ten images. Unfortunately, it is impossible for me to get more images of the scene, since the images are taken by ten fixed cameras that I cannot access. Is there a way to use the script on such a small number of images, for example by extending the search/reducing the expected quality of the outcome etc.?

**Comment by akkkb** [comment]
You can first try by modifying aabb_scale 8,4,2,1.
Can try  default --colmap_matcher i.e., sequential. 
I hope it should resolve the issue.

**Comment by PascalHbr** [comment]
Unfortunately, this does not fix the issue :/

**Comment by jbb-kryo** [comment]
You need a minimum of 50 images for a successful match. Personally, I would recommend a minimum of 80-100 for decent results.

**Comment by PascalHbr** [comment]
Ah I see, thanks! So what if I request the camera positions - can I expect good results regarding the NERF performance from just 10 images?

**Comment by jbb-kryo** [comment]
Nope. You need a lot more photos for a decent training. Training input is key and too few reference points and it cannot make the calculations it needs to work. I suppose theoretically if you had 10x photos which were super-wide CCD derived at 10k+ resolution, with serious color depth, and massive resolution, and they overlapped at their edges significantly - it could potentially work.

**Comment by ishipachev** [comment]
Make sure that you are using a proper camera model when you do reconstruction.

**Comment by jexiaong** [comment]
I have 245 images but I still get this error... does anyone have any tips?

**Comment by AvishkarArjan** [comment]
Hey anyone find any soln to this yet ? M still having this issue
